apps/omg_eth/test/support/dev_geth.ex

```diff
@@ -36,27 +38,75 @@ defmodule OMG.Eth.DevGeth do
     geth = ~s(geth --miner.gastarget 7500000 \
             --nodiscover \
             --maxpeers 0 \
-            --miner.gasprice \"10\" \
-            --datadir /data/ \
+            --miner.gasprice "10" \
             --syncmode 'full' \
             --networkid 1337 \
             --gasprice '1' \
             --keystore #{keystore} \
             --password /tmp/geth-blank-password \
-            --unlock \"0,1\" \
+            --unlock "0,1" \
             --rpc --rpcapi personal,web3,eth,net --rpcaddr 0.0.0.0 --rpcvhosts='*' --rpcport=8545 \
             --ws --wsaddr 0.0.0.0 --wsorigins='*' \
             --allow-insecure-unlock \
             --mine --datadir #{datadir} 2>&1)
-    geth_pid = launch(geth)
+    pid = launch(geth)
 
     {:ok, :ready} = WaitFor.eth_rpc(20_000)
 
-    on_exit = fn -> stop(geth_pid) end
+    on_exit = fn -> stop_geth(pid) end
 
     {:ok, on_exit}
   end
 
+  def ready?(pid) do
+    GenServer.call(pid, :ready?)
+  end
+
+  def stop_geth(pid) do
+    GenServer.cast(pid, :stop)
+  end
+
+  def start_link(cmd) do
+    GenServer.start_link(__MODULE__, cmd)
+  end
```

Comment:
these could be private?

Reply:
@InoMurko I made them private and fixed tests